inference-engine/src/vpu/graph_transformer/src/frontend/parse_data.cpp

```diff
@@ -116,21 +116,6 @@ void FrontEnd::parseInputAndOutputData(const Model& model) {
             descriptor,
             ieBlobContent(ieBlob, descriptor.type()));
 
-        // User might ask to return the output from Const layer.
-        if (const auto vpuOutData = getVpuData(ieData)) {
-            env.log->trace("The constant %s is network output", vpuData);
-
-            IE_ASSERT(vpuOutData->usage() == DataUsage::Output);
-
-            _stageBuilder->addCopyStage(
-                model,
-                formatString("%s@return-const", vpuData->name()),
-                nullptr,
-                vpuData,
-                vpuOutData,
-                "parseInputAndOutputData::const");
-        }
-
```

Comment:
Why do you need to remove this?

Reply:
`processTrivialCases` now covers this case as well, so this would be a logic duplication.